db/migrate/20200214224815_index_accounts_on_sec_id.rb

```diff
@@ -0,0 +1,8 @@
+class IndexAccountsOnSecId < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    # TODO: should this be a unique index instead?
```

Comment:
I don't think so - for the same reason that the icn/edipi indexes are not unique, we could have somebody with two credentials mapping to the same MVI identity and I think those would result in duplicate SECIDs.